### ISSUE TITLE
Problem with Mastercard statement (PDF to CSV)

### ISSUE THREAD

**Issue by Deasmhuin** [tabulapdf/tabula#841]
Tried to extract table from Mastercard statement (PDF).  Instead of text I get  what looks like PDF control codes.  Would like csv of table
[p0817.pdf](https://github.com/tabulapdf/tabula/files/1951190/p0817.pdf)

Appreciate any help./ Thanks

**Comment by jazzido** [comment]
@Deasmhuin, yes, it's not advisable to post those documents publicly. On the other hand, it's likely that your PDF has bad glyph-to-character mappings. Try copying from Acrobat Reader or other PDF viewing app and pasting onto a text editor. Do you also see gibberish instead of the actual text? If so, and unless you're willing to do some serious, low-level PDF debugging, OCR is your only option.

See this Stack Overflow answer for more information: http://stackoverflow.com/questions/12184304/if-pdf-copypaste-gives-garbage-text-what-is-wrong-with-the-pdf-source-code
